src/timer/CountDownTimer.test.jsx

```diff
@@ -296,13 +296,14 @@ describe('ExamTimerBlock', () => {
 
   const timesToTest = {
     // Because times are rounded down, these values are 60 seconds off
```

Comment:
wasn't sure whether or not to remove this comment yet, wanted to make sure it was okay that these values changed